Sources/TuistGenerator/Settings/DefaultSettingsProvider.swift

```diff
@@ -139,16 +139,34 @@ public final class DefaultSettingsProvider: DefaultSettingsProviding {
             essentialKeys: DefaultSettingsProvider.essentialTargetSettings,
             newXcodeKeys: DefaultSettingsProvider.xcodeVersionSpecificSettings
         )
+        let mergeableSettings = mergeableSettings(for: target, configuration: buildConfiguration)
         var settings: SettingsDictionary = [:]
         settingsHelper.extend(buildSettings: &settings, with: targetDefaultAll)
         settingsHelper.extend(buildSettings: &settings, with: additionalTargetDefaults)
         settingsHelper.extend(buildSettings: &settings, with: targetDefaultVariant)
+        settingsHelper.extend(buildSettings: &settings, with: mergeableSettings)
         settingsHelper.extend(buildSettings: &settings, with: projectOverridableTargetDefaultSettings(for: project))
         return settings.filter(filter)
     }
 
     // MARK: - Private
 
+    private func mergeableSettings(for target: Target, configuration: BuildConfiguration) -> SettingsDictionary {
+        var settings: SettingsDictionary = [:]
+        if target.mergeable {
+            settings["MAKE_MERGEABLE"] = .string("YES")
```

Comment:
nit: I think you can just do 
```suggestion
            settings["MAKE_MERGEABLE"] = "YES"
```
same below

Reply:
fixed